rust/src/utils.rs

```diff
@@ -675,6 +675,7 @@ impl BigInt {
             return None;
         }
         match u64_digits.len() {
+            0 => Some(to_bignum(0)),
```

Comment:
unrelated to this PR, this was just a bug similar to the issue we had for serializing 0 as `BigInt` before.